src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3615,39 +3615,30 @@
      * <p>For example:</p>
      *
      * <pre>
-     *     public class Blah&lt;A, B&gt;
-     *     {
-     *     }
+     * public class MyClass&lt;A, B&gt; {
```

Comment:
`public` is redundant here, but without it there will be a dangling `MODIFIERS` node in the AST. I'm ok to keep this modifier.

Reply:
Okay!!